src/PathMatch/PathMatcher.php

```diff
@@ -0,0 +1,280 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Files\PathMatch;
```

Comment:
```suggestion
namespace Yiisoft\Files\Matcher;
```
It would be better :)

Reply:
Or `Yiisoft\Files\PathMatcher`?

Reply:
I'm ok.

Reply:
Done.